web/src/routes/PoisPage.tsx

```diff
@@ -184,28 +181,29 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
     )
   }
 
+  const switchFeature = (step: number) => {
+    const featureIndex = featureLocations.findIndex(
+      (poi: PoiFeature) => poi.properties.urlSlug === currentFeature?.properties.urlSlug
+    )
+    const updatedIndex = updateFeatureIndex(step, featureLocations.length, featureIndex)
```

Comment:
Could you also update the `updateFeatureIndex` method? I think it is clearer if the step is typed literally as it can just be -1 or 1 otherwise the method does not work as expected. Also I'd suggest naming it nextFeatureIndex, as update inclines that something is changed in the method. :)

```typescript
    const nextFeatureIndex = (step: -1 | 1, arrayLength: number, currentIndex: number) { /* ... */ }
```